src/jsx.d.ts

```diff
@@ -2,6 +2,7 @@ import {VNode, VNodeData} from './vnode';
 
 declare global {
   namespace JSX {
+    type Element = VNode;
```

Comment:
What's this for, please?

Reply:
As the intrinsic interface it says what a jsx element type is. It’s a Typescript built in. 

Essentially any jsx node has its inferred type as JSX.Element. With a type alias. 

`const node = <div />` will have it’s inferred type as `VNode`